lighthouse-viewer/app/index.html

```diff
@@ -35,6 +35,11 @@ <h1 class="viewer-placeholder__heading">Lighthouse Report Viewer</h1>
       or <a href="https://web.dev/measure">web.dev/measure</a>.</div>
       <input type="url" class="viewer-placeholder__url js-gist-url"
              placeholder="Enter or paste Gist URL">
+
+      <br>
```

Comment:
add more breaks or more judicious max-width on a `<p>` perhaps? this doesn't jive so well with the other calculated line breaks we have (though explicit <br> for web.dev/measure isn't exactly an iron clad decision IMO either)

![image](https://user-images.githubusercontent.com/2301202/127940262-a086e72a-8682-4a83-958e-b24583abe4c1.png)